File: core/identity-service/src/main/java/com/gov/crypto/IdentityServiceApplication.java
package com.gov.crypto.identityservice;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.autoconfigure.domain.EntityScan;
import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication(scanBasePackages = "com.gov.crypto")
@EntityScan("com.gov.crypto.model")
@EnableJpaRepositories("com.gov.crypto.repository")
@EnableScheduling
public class IdentityServiceApplication {

    public static void main(String[] args) {
        SpringApplication.run(IdentityServiceApplication.class, args);
    }
}

File: backend/identity-service/src/main/kotlin/com/gov/crypto/identityservice/IdentityServiceApplication.kt
package com.gov.crypto.identityservice

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan

@SpringBootApplication
@ComponentScan(basePackages = ["com.gov.crypto"])
class IdentityServiceApplication

fun main(args: Array<String>) {
    runApplication<IdentityServiceApplication>(*args)
}
